packages/strapi-plugin-i18n/services/localizations.js

```diff
@@ -16,17 +16,23 @@ const assignDefaultLocale = async data => {
 };
 
 /**
- * Create default localizations for an entry if it isn't defined yet
+ * Syncronize related localizations from a root one
  * @param {Object} entry entry to update
  * @param {Object} options
  * @param {Object} options.model corresponding model
  */
-const addLocalizations = async (entry, { model }) => {
-  if (isNil(entry.localizations)) {
-    const localizations = [{ locale: entry.locale, id: entry.id }];
-    await strapi.query(model.uid).update({ id: entry.id }, { localizations });
+const syncLocalizations = async (entry, { model }) => {
+  if (Array.isArray(entry.localizations)) {
+    const newLocalizations = entry.localizations.concat({ id: entry.id });
```

Comment:
Can't you map to prop('id') here instead of doing it every time within the `updateLocalization` below?